### ISSUE TITLE
App crashes in Sandbox on windows 10 release 21H1

### ISSUE THREAD

**Issue by Rick6561** [sandboxie-plus/Sandboxie#940]
I have tried several versions including 3 from Classic and the latest plus release.

Run Zwift game in sandbox. App starts, gets to a point and crashes silently. Sandbox is fine. I have read other posts where previous releases of Windows 10 exhibit the same behavior.

As a note, different workstation with Windows 10 20H2 runs the same game application without issue in the sandbox.

This workstation has Windows 10 21H1. Zwift runs fine outside of the sandbox.

Thx
Rick

**Comment by isaak654** [comment]
With Plus 0.8.2, you need to enable 'Trace Log' tab by opening 'Options' menu -> 'Trace Logging'.
Once enabled, you can reproduce your issue.
After reproducing it, just right click -> 'Copy Panel' to copy the full log and post it here:
![trace_log](https://user-images.githubusercontent.com/12372772/113529350-793af380-95c3-11eb-8d48-f9aab655b37a.png)

**Comment by DavidXanatos** [comment]
can i download that gane for free someware for testing?

**Comment by Rick6561** [comment]
Yes there is a 7 day free trial: https://www.zwift.com/. Create an account and then download the game app.

The app will start, ask for a login and then eventually get to the a screen where you pair sensors (see attached)
![pairsensors](https://user-images.githubusercontent.com/86484439/123644220-2acc7800-d7f3-11eb-890f-096defaf28a4.jpg)

In the failure mode this screen will present itself for a few moments and then close unceremoniously. In the working mode that screen would sit there waiting to pair either an ANT or Bluetooth sensor. 

I'm guessing you will not have such sensors so if the paring screen does not fail you can choose to just watch. A warning will come up telling you that you haven't paired with anything, that's fine, it will pick a rider and show you that rider in real time just as an observer. If you can get to that point I suspect all is well.

Running it outside of Sandboxie should allow you to move through to the Just Watch mode as described in the paragraph above.

I have a log but unfortunately not with me at the moment. Although my read of the log doesn't present anything noteworthy. I will get the log to you later this evening.

Thx
Rick

**Comment by Rick6561** [comment]
I get a popup sandboxie notification window message: ZwiftLauncher.exe (5604): SBIE2205 Service not implemented:  ImpersonateNamedPipe

Attached is trace log
[sandboxielog.txt](https://github.com/sandboxie-plus/Sandboxie/files/6742028/sandboxielog.txt)

**Comment by isaak654** [comment]
Thank you for attaching the log.
Please add these lines on your sandbox, and let me know if something changes on your side:
```
OpenPipePath=\Device\NamedPipe\ZwiftLauncherrickb*
OpenIpcPath=*\BaseNamedObjects*\ZwiftLauncherrickb
```

**Comment by Rick6561** [comment]
Away from workstation for a week... still no joy. Immediately get the popup indicating SBIE2205 Service not implemented: ImpersonateNamedPipe

The entries are not present on the workstation with the other windows version.

Below is ini:
[zwift]

Enabled=y
ConfigLevel=9
Template=OpenSmartCard
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00ffff,ttl,6
BoxNameTitle=y
OpenPipePath=\Device\NamedPipe\ZwiftLauncherrickb*
OpenIpcPath=*\BaseNamedObjects*\ZwiftLauncherrickb

**Comment by Rick6561** [comment]
Sorry I was fiddling with the entries, the entries as given produced the popup as well.
[zwift]

Enabled=y
ConfigLevel=9
Template=OpenSmartCard
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00ffff,ttl,6
BoxNameTitle=y
OpenPipePath=\Device\NamedPipe\ZwiftLauncherrickb*
OpenIpcPath=*\BaseNamedObjects*\ZwiftLauncherrickb

**Comment by robert-budde** [comment]
Hi! I would like to raise this topic as it affects me too.

Maybe I can add some more info: I CAN run Zwift in Sandboxie with my Bluetooth stick UNPLUGGED. I can, by the way, safely plug in the BT stick after the sensor connecttion screen mentioned above. So quite likely it has something to do with BT discovery?

BR
Robert
